{
  "schema_version": "1.4.0",
  "id": "GHSA-jx23-r23x-qc23",
  "modified": "2022-08-28T00:00:28Z",
  "published": "2022-08-26T00:03:30Z",
  "aliases": [
    "CVE-2020-27802"
  ],
  "details": "An floating point exception was discovered in the elf_lookup function in p_lx_elf.cpp in UPX 4.0.0 via a crafted Mach-O file.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
    }
  ],
  "affected": [],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-27802"
    },
    {
      "type": "WEB",
      "url": "https://github.com/upx/upx/issues/393"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-119",
      "CWE-369"
    ],
    "severity": "MODERATE",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2022-08-25T20:15:00Z"
  }
}